src/RichTextEditor/utilities/toolbarHelpers.js

```diff
@@ -27,6 +27,11 @@ export const toggleBlock = (editor, format) => {
   Transforms.unwrapNodes(editor, { match: n => isListItem(n.type), split: true });
   Transforms.unwrapNodes(editor, { match: n => LIST_TYPES.includes(n.type), split: true });
 
+  if (format === 'paragraph' || format.startsWith('heading')) {
```

Comment:
This will allow to turn paragraph into heading and vice versa, is that correct?

Reply:
correct!